README.md

```diff
@@ -120,6 +123,7 @@ For a list of options, see the configuration parameters below the code block. 
 | logzio_url | **Default**: `https://listener.logz.io:8071` <br> Listener URL and port. <br> Replace `<<LISTENER-HOST>>` with your region's listener host (for example, `listener.logz.io`). For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html). |
 | logzio_type | **Default**: `logzio-fluent-bit` <br> The [log type](https://docs.logz.io/user-guide/log-shipping/built-in-log-types.html), shipped as `type` field. Used by Logz.io for consistent parsing. Can't contain spaces. |
 | logzio_debug | **Default**: `false` <br>  Set to `true` to print debug messages to stdout. |
+| id           | **Default**: `1` <br> Output name. Mandatory when using multiple outputs.                                                                                                                                                                                                                                                |
```

Comment:
Can you change the default value to be something like logzio_output1. 1 is to generic and maybe there is this kind of id in the system already